draft-ietf-quic-multipath.md

```diff
@@ -541,9 +541,14 @@ send path control information and specifically acknowledge packets belonging to
 packet number space.
 
 A path in the "Validating" state performs path validation as described
-in {{Section 8.2 of QUIC-TRANSPORT}}. An endhost should not send
+in {{Section 8.2 of QUIC-TRANSPORT}}. An endhost SHOULD NOT send
 non-probing frames on a path in "Validating" state, as it has no
-guarantee that packets will actually reach the peer.
+guarantee that packets will actually reach the peer, with an exception
+of the ACK_MP frame. An endhost MAY bundle the ACK_MP frame with the
```

Comment:
IIUC, this text builds on the fact that ACK_MP is a non-probing frame (because ACK is a non-probing frame), and states that even though ACK_MP is a non-probing frame it can be sent in a probe packet.

I wonder if it would be less confusing to simply state that ACK_MP is a probing frame?

Reply:
not sure what to do here but let's further discuss this in issue #226 (and #50)

Reply:
@kazuho I agree with you. I think making ACK_MP a probing frame is the easiest fix.